astropy/table/table_helpers.py

```diff
@@ -168,16 +168,16 @@ class ArrayWrapper:
     """
     info = ArrayWrapperInfo()
 
-    def __init__(self, data):
-        self.data = np.array(data)
+    def __init__(self, data, copy=True):
+        self.data = np.array(data, copy=False)
```

Comment:
Shouldn't this be `copy=copy`?

Reply:
Yes, of ccourse! Silly me. Fixed (and rebased so now everything should pass)